jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Currently there is no config dir: https://github.com/start-jsk/jsk_apc/tree/master/jsk_arc2017_common